packages/x-data-grid/src/hooks/features/virtualization/gridVirtualizationSelectors.ts

```diff
@@ -54,17 +54,22 @@ export const gridRenderContextSelector = createSelector(
   (state) => state.renderContext,
 );
 
+const firstColumnIndexSelector = createRootSelector(
+  (state: GridStateCommunity) => state.virtualization.renderContext.firstColumnIndex,
+);
+const lastColumnIndexSelector = createRootSelector(
+  (state: GridStateCommunity) => state.virtualization.renderContext.lastColumnIndex,
+);
+
 /**
  * Get the render context, with only columns filled in.
  * This is cached, so it can be used to only re-render when the column interval changes.
  * @category Virtualization
  * @ignore - do not document.
  */
 export const gridRenderContextColumnsSelector = createSelectorMemoized(
-  (apiRef: RefObject<GridApiCommunity>) =>
-    apiRef.current.state.virtualization.renderContext.firstColumnIndex,
-  (apiRef: RefObject<GridApiCommunity>) =>
-    apiRef.current.state.virtualization.renderContext.lastColumnIndex,
```

Comment:
Actually this might be a breaking change, the create selector functions only accept other selectors as arguments. I might need to handle non-internal selectors differently.

Reply:
Yes, every selector passed into `createSelector(Memoized)` now needs to handle that the first param is `apiRefOrState`. Only these two inline selectors weren't doing it.